layouts/partials/header/custom.html

```
@@ -0,0 +1,5 @@
+<!-- Web standards - manifest and icons -->
+<link rel="manifest" href="/manifest.json">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)